distribution/docs/src/main/resources/_contents/_endpoints/catalog-rest-endpoint-contents.adoc

```diff
@@ -1,3 +1,9 @@
+:title: Catalog REST Endpoint
+:type: endpoint
+:status: published
+:filename: catalog-rest-endpoint-contents.adoc
```

Comment:
Can you use `content.file` instead?

http://jbake.org/docs/2.5.1/#page_post_custom